### ISSUE TITLE
buildnativeoperations.sh detects non-existing cmake3 in MSYS2

### ISSUE THREAD

**Issue by ynpandey** [deeplearning4j/libnd4j#360]
I am trying to build libnd4j DLL on Windows 7. buildnativeoperations.sh was executing the if block for cmake3, even though I only had cmake in my MSYS2. Commenting out the cmake3 block fixed "cmake3: eval: ./buildnativeoperations.sh 288: not found" error.
```
export CMAKE_COMMAND="cmake"
#if which cmake3 &> /dev/null; then
# export CMAKE_COMMAND="cmake3"
#fi
```
Could you please look into the problem causing this issue.

Thank you,
Yogendra

**Comment by raver119** [comment]
Have you followed instructions listed here: https://github.com/deeplearning4j/libnd4j/blob/master/windows.md ?

**Comment by ynpandey** [comment]
Yes. I have followed the instructions on the mentioned page. I used command `pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-extra-cmake-modules make pkg-config grep sed gzip tar mingw64/mingw-w64-x86_64-openblas mingw-w64-x86_64-lz4` to get the required mingw packages.

**Comment by saudet** [comment]
What does `which cmake3` return on your machine?

**Comment by ynpandey** [comment]
HXXXXXX@WKSXXXXXX:/d/workspace/workspace-git/dl4j/libnd4j::which cmake3
which: no cmake3 in (C:/maven-3.3.9/bin:/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/c/Windows/System32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0/:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)

**Comment by saudet** [comment]
So, it fails, and the if branch cannot be taken... That should be alright, right?

**Comment by ynpandey** [comment]
Ideally, it should fail. But somehow for me it was executing the if branch, making CMAKE_COMMAND cmake3.
